ThermofluidStream/Interfaces/SISOFlow.mo

```diff
@@ -30,12 +30,12 @@ partial model SISOFlow "Base Model with basic flow eqautions for SISO"
   Outlet outlet(redeclare package Medium=Medium)
     annotation (Placement(transformation(extent={{80,-20},{120,20}})));
 
-SI.MassFlowRate m_flow(stateSelect=m_flowStateSelect) = inlet.m_flow "mass flow through component";
-
-// changing pressure calculation
-SI.Pressure dr_corr; // delta = out - in
-SI.Pressure dp; // delta = out - in
+  SI.MassFlowRate m_flow(stateSelect=m_flowStateSelect) = inlet.m_flow
+    "Mass flow through component";
```

Comment:
Intended line break?

Reply:
More or less. But I agree there is no rule for maximum line length. So we can revert this line break.

Reply:
Okay, I was just wondering if Dymola once again caused some unwanted line breaks :)

Reply:
No no, it was really me. :-|